SharpGrad/Value.cs

```diff
@@ -5,7 +5,7 @@ public class Value
         private readonly double _value;
         private List<Value> _children = new List<Value>();
         private Action _backward = () => { };
-        private double _grad;
+        private double _grad = 0;
```

Comment:
This is a comment from GitHub on line 8.

Reply:
<!-- Comment from Crocodile -->
Reply back from Crocodile

Posted from [Crocodile 🐊](https://app.crocodile.dev/reviews/rwPWqdvJHMJe)